plot from NWB files

luiztauffer opened the pull request:
I added a file 'plot_from_nwb.py' that allows the use of the visualization tools from img_pipe but reading directly from NWB files. The changes contain:

- the 'plot_from_nwb.py' file, located at '/plotting' directory
- the Jupyter Notebook tutorial 'img_pipe_plot_from_nwb.ipynb', located at '/tutorials' directory
- Five image files for the tutorial, located at '/gallery' directory
- changed 1 line at the '__init__.py' file from '/plotting', to load the new function
- added the dependency 'nwbext_ecog' to be installed from pip, at 'environment_py35.yml'

Commit:
plot from nwb

Commit:
tutorial

Commit:
update

Commit:
env_update

Commit:
update

Commit:
update

Commit:
update

Commit:
update function

Commit:
update function

Commit:
update

bendichter committed:
Update environment_py35.yml

bendichter committed:
Update environment_py27.yml

bendichter committed:
Update environment_py27.yml

luiztauffer committed:
Update img_pipe_plot_from_nwb.ipynb

luiztauffer committed:
Update img_pipe_plot_from_nwb.ipynb

bendichter commented:
@luiztauffer let's try to get the CI to pass. Could you add pynwb as a dependency?

Commit:
add nwbext_ecog to requirements

Commit:
add nwbext_ecog

Commit:
nwbext_ecog

Commit:
test env build

Commit:
envpy27 change

Commit:
return setup to previous version

bendichter committed:
update nwbext_ecog

bendichter committed:
Update environment_py27.yml

bendichter committed:
Update environment_py27.yml

bendichter committed:
works with py36

moved mayavi from conda to pip
removed subject path calls from plotting in NWB
added yaml for py36

bendichter committed:
Merge remote-tracking branch 'luiztauffer/master' into luiz_master

# Conflicts:
#	tutorials/img_pipe_plot_from_nwb.ipynb

bendichter committed:
lock dependencies for py36

bendichter commented:
@luiztauffer can you try installing from environment_py36.yml? It's working on my machine but I can't get it to work on the continuous integration

Commit:
changes dependencies
adds channels

luiztauffer commented:
Doing the following made the conda install pass:
- Comment out some locks e.g.  - curl=7.64.1 #=ha441bb4_0
- Comment out 'appnope', which is only available for mac. 
- Added channels conda-forge and statiskit

Pip is still failing though.
